src/Modal.js

```diff
@@ -142,11 +142,11 @@ const Modal = React.createClass({
 
   render() {
     let {
-        className
-      , children
-      , dialogClassName
-      , animation
-      , ...props } = this.props;
+      className,
```

Comment:
I got linting errors from that one (and the lint errors stopped the build) so I changed it to comply. Like I said in the PR Comment, this PR is doing way too much ;)